### PR TITLE
VW MQB: Cleanup stock ACC button handling

### PR DESCRIPTION
**Pull request by jyoung8607**
Remove my ugly cancel/resume spam implementation and replace in the style of @robbederks's nice Tesla implementation.

Passes tests but haven't driven yet.

### CODE REVIEW

File: selfdrive/car/volkswagen/carstate.py [hunk 1]
```diff
@@ -114,26 +115,20 @@ def update(self, pt_cp, cam_cp, ext_cp, trans_type):
...
-    self.graHauptschalter = pt_cp.vl["GRA_ACC_01"]["GRA_Hauptschalter"]
-    self.graTypHauptschalter = pt_cp.vl["GRA_ACC_01"]["GRA_Typ_Hauptschalter"]
-    self.graButtonTypeInfo = pt_cp.vl["GRA_ACC_01"]["GRA_ButtonTypeInfo"]
-    self.graTipStufe2 = pt_cp.vl["GRA_ACC_01"]["GRA_Tip_Stufe_2"]
-    # Pick up the GRA_ACC_01 CAN message counter so we can sync to it for
-    # later cruise-control button spamming.
-    self.graMsgBusCounter = pt_cp.vl["GRA_ACC_01"]["COUNTER"]
+    self.gra_stock_values = pt_cp.vl["GRA_ACC_01"]
+    buttonEvents = []
+    for button in MQB_BUTTONS:
```
Comment (sshane) [inline selfdrive/car/volkswagen/carstate.py:123]: Nice clean up! Can we use `create_button_event` here?

Comment (jyoung8607) [reply selfdrive/car/volkswagen/carstate.py:123]: To summarize our DMs, I'll keep this in draft until you settle on the best way to merge the Tesla/global button state implementations, and then I'll sync up with that.

**Comment by sunnyhaibin** [comment]
This is a great cleanup. Nice job!

File: selfdrive/car/volkswagen/carstate.py (hunk 1 above)
Comment (sshane) [reply selfdrive/car/volkswagen/carstate.py:123]: Let's merge this with Tesla's implementation as VW uses different bits for its buttons too. We can clean up and unify for all cars in another PR. Is this ready for review?

Comment (jyoung8607) [reply selfdrive/car/volkswagen/carstate.py:123]: It's complete, but I haven't driven it yet. Give me the weekend to make sure cancel spam and resume spam are solid.

**Comment by jyoung8607** [comment]
Both cancel and resume LGTM after minor bug fix: `3cfdec54aa035f3f|2022-07-18--17-01-20`. Ready for review.

**Comment by sshane** [comment]
Thanks for the PR!

**Pull request merged**